unpackaged/config/trial/objects/Contact.object

```diff
@@ -47,6 +47,7 @@
     </compactLayouts>
     <fields>
         <fullName>hed__Citizenship__c</fullName>
+        <description>The person's country of citizenship. If this student has dual citizenship, indicate the second country of citizenship in the Dual Citizenship field.</description>
```

Comment:
@MatthewBlanski should we be added help text too?

Reply:
No, I don't think so. It'll deploy cleanly without it. We now get:

```
2020-06-19 17:24:01: [Failed]: Update of Profile Admin: Error on line 23, col 24: In field: layout - no Layout named hed__Behavior_Response__c-K12 Kit Behavior Response Layout found

Error: Could not process MDAPI response: Update of Profile Admin: Error on line 23, col 24: In field: layout - no Layout named hed__Behavior_Response__c-K12 Kit Behavior Response Layout found
```
@MatthewBlanski I can force-merge #250, or we can include the commit in this PR.